### PR TITLE
updates

### PR DESCRIPTION
**Pull request by aanchalcoder18**
updated redesign for homepage after kahlil's feedback

### CODE REVIEW

**Comment by vinhnguyen200891337** [comment]
1.The layout in homepage looks weird. You need some kind of text or button in the Discover Activism that makes you banner. 
2.The New to ThinQ.tv? need to move to the right a little bit. 
3.The countdown clock's text looks kinda bland, you may need to add some color to it.
4. Also there is too much space between the drop in button in the dropin hours pic. When I add both dropin and problem solving button, the latter goes into the picture below. 
5. Also the caption underneath the pics need to be realigned to the right or something so that it looks better.
6. The Monthly View of the calendar may need to be smaller so that it does not stretch across the screen.
7.The footer text are not aligned properly. The footer in the Control Panel (specifically the Rewards. Sales, Blocked Users) has too much bottom padding.

**Comment by RogelioMart** [comment]
TL/DR: I do not approve of this pull request being merged into master.

Things they fixed:

The countdown clock doesn't overlap anymore.

The monthly calendar now displays their appropriate chatroom topics there is no mix up with conversations and drop in.

Problems the homepage still has:

Found that when creating a reward in donations it does not let you upload an image instead you have to set it to buy upload the image and then change it to donation to upload an image with the donation.
I doubt that this is due to the pull request  but it will confuse the users when they try to make rewards.

The footer is misaligned.

Screenshot from 2020-08-10 17-48-02


the text under conversations and drop in hours is misaligned.

Screenshot from 2020-08-10 19-00-39


Why doesn't the drop in button display information about who's hosting and that it is live right now also it seems horizontally misaligned with the join conversation button.

Screenshot from 2020-08-10 19-02-40


when going to the events show page of a conversation or drop in hour it doesn't display the banner.

Screenshot from 2020-08-10 19-01-25


When trying to set ups a drop in hour at the same time as another one the error message still displays the phrase activism hall. Also I don't think the second smaller message is necessary.

Screenshot from 2020-08-10 19-10-48


Why countdown clocks seem horizontally misaligned and the one for drop in hours doesn't display the days, hours, minutes etc.

Screenshot from 2020-08-10 19-27-00


Personal opinions: It is not obvious for me how to get to the drop in hours list. It just seems like a highlighted word but it doesn't seem obvious to me that I should click it.

Screenshot from 2020-08-10 19-16-15

**Comment by aanchalcoder18** [comment]
This is supposed to be a first cut of the homepage - so there are a lot of things that still needed to be worked on.

**Comment by ece106** [comment]
Definitely not ideal - an some are issues with other portions of the site that should be dealt with - but still possibly an improvement over the existing home page.
The problem I have is ... Why do the large maroon fonts have serifs???
Also, that 2nd countdown clock absolutely needs to be fixed before merging into master.
